domains/client/domain-operator/src/domain_worker_starter.rs

```diff
@@ -118,27 +118,8 @@ pub(super) async fn start_worker<
             consensus_block_import_throttling_buffer_size,
         );
 
-    if !is_authority {
-        info!("🧑‍ Running as Full node...");
-        drop(new_slot_notification_stream);
-        drop(acknowledgement_sender_stream);
-        while let Some(maybe_block_info) = throttled_block_import_notification_stream.next().await {
-            if let Some(block_info) = maybe_block_info {
-                if let Err(error) = bundle_processor
-                    .clone()
-                    .process_bundles((block_info.hash, block_info.number, block_info.is_new_best))
-                    .instrument(span.clone())
-                    .await
-                {
-                    tracing::error!(?error, "Failed to process consensus block");
-                    // Bring down the service as bundles processor is an essential task.
-                    // TODO: more graceful shutdown.
-                    break;
-                }
-            }
-        }
-    } else {
-        info!("🧑‍🌾 Running as Operator...");
+    if let Some(operator_id) = maybe_operator_id {
+        info!("🧑‍🌾 Running as Operator[{operator_id:?}]...");
```

Comment:
If it is just a number, should probably implement `std::fmt::Display` in addition to `std::fmt::Debug`. Also we need a different emoji, farmer is not appropriate here, 👷 or 🧑‍🏭 or 🧑‍💼 seem relevant.